.github/workflows/publish.yaml

```diff
@@ -26,7 +28,7 @@ jobs:
           message: '${{ github.event.head_commit.message }}'
           expo_token: ${{ secrets.EXPO_TOKEN }}
 
-  build-android:
+  build-and-submit-android:
```

Comment:
there is a way to trigger builds and submits for both platforms, is that something we'd like to do? that sure would make it easy to keep the build numbers in sync, but comes back to the conversation of do we want to have the two apps be in sync when it comes to store submissions? 

https://docs.expo.dev/build/building-on-ci/

Reply:
As long as we have a trivially easy way to derive the repo state from some combination of runtime version, build version and update UUID, I don't know that having the same literal build number is that important. The more important part would be making sure that Android and iOS are running the same code.

Reply:
agreed, and we should be able to do that easily. 

do you think it would be nice to have 2 separate github action jobs to submit the apps per platform? or merging it into one would be cleaner?

Reply:
One for each and a third to do both could give us the best of all worlds

Reply:
With OTA updates, the installed app version (as seen by the app stores) is probably less significant than the bundle version / git SHA that the user is running. 

I might be running an ancient app binary, but as long as Expo keeps downloading new JS through OTA updates, I'm staying fresh.